conf.d/hydro.fish

```diff
@@ -58,41 +58,43 @@ function _hydro_prompt --on-event fish_prompt
 
     command kill $_hydro_last_pid 2>/dev/null
 
-    fish --private --command "
-        ! command git --no-optional-locks rev-parse 2>/dev/null && set $_hydro_git && exit
-
-        set branch (
-            command git symbolic-ref --short HEAD 2>/dev/null ||
-            command git describe --tags --exact-match HEAD 2>/dev/null ||
-            command git rev-parse --short HEAD 2>/dev/null |
-                string replace --regex -- '(.+)' '@\$1'
-        )
-
-        test -z \"\$$_hydro_git\" && set --universal $_hydro_git \"\$branch \"
-
-        ! command git diff-index --quiet HEAD 2>/dev/null ||
-            count (command git ls-files --others --exclude-standard) >/dev/null &&
-            set info \"$hydro_symbol_git_dirty\"
-
-        for fetch in $hydro_fetch false
-            command git rev-list --count --left-right @{upstream}...@ 2>/dev/null |
-                read behind ahead
-
-            switch \"\$behind \$ahead\"
-                case \" \" \"0 0\"
-                case \"0 *\"
-                    set upstream \" $hydro_symbol_git_ahead\$ahead\"
-                case \"* 0\"
-                    set upstream \" $hydro_symbol_git_behind\$behind\"
-                case \*
-                    set upstream \" $hydro_symbol_git_ahead\$ahead $hydro_symbol_git_behind\$behind\"
+    if ! contains $hydro_ignored_git_paths
```

Comment:
How about we invert the logic here so that we just exit the process and also to prevent the indentation?